ui/src/util/convertUTCDateToLocalDate.js

```diff
@@ -1,3 +1,8 @@
+// The Aleph API returns dates as ISO strings without timezone
+// designators. When initializing new `Date` objects, those dates
+// will by default be initialized with the local timezone of the
+// user agent.
 export default function convertUTCDateToLocalDate(date) {
-  return new Date(`${date} UTC`);
+  const dateObj = new Date(date);
+  return new Date(dateObj.getTime() - dateObj.getTimezoneOffset() * 60 * 1000);
```

Comment:
There was a similar implementation before that Kirk changed in https://github.com/alephdata/aleph/commit/f30763ea5924222ec21ac47d95e0592a45c6fb92.

But if I understand correctly, the reason to move away from the old implementation was that it would fail if the day was different in the local vs UTC timezone.

This implementation shouldn’t have this issue, as it uses timestamps.